encode/vp9enc.c

```diff
@@ -814,8 +819,62 @@ vp9enc_update_reference_list()
    }
 }
 
+static
+void vp9enc_set_refreshparameter_for_svct_2layers(VAEncPictureParameterBufferVP9 *pic_param, int current_frame, int *is_golden_refreshed)
+{
+  //Pattern taken from libyami
```

Comment:
Though // is valid for comments in modern C, for sake of coherence, perhaps would be better to stick to the old /* */

Reply:
I fixed this for my part of the code, but there are other occurrences of // as well in vp9enc.c